include/h2o/serverutil.h

```diff
@@ -38,6 +38,12 @@
 #define EX_CONFIG 78
 #endif
 
+/**
+ * environment variable name for listening socket's addr and fd
+ * example: 127.0.0.1:80=3;/tmp/sock=4
+ */
+#define SERVER_STARTER_ENV_NAME "SERVER_STARTER_PORT"
```

Comment:
Thank you for the PR.

How about renaming the macro to `SERVER_STARTER_PORT`?

I think it makes sense to define the name as a macro (since by doing so we can detect typos), but introducing another name for an environment name seems like adding a layer of confusion to me.

Other that I think the PR is ready to merge.

Reply:
ok, will do it later.